com.bluejeans.BlueJeans.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: com.bluejeans.BlueJeans
+base: io.atom.electron.BaseApp
```

Comment:
Not Electron 2?

Reply:
Does not seem to work when using the Electron 2 base app:
`/app/extra/bluejeans: error while loading shared libraries: libgtk-x11-2.0.so.0: cannot open shared object file: No such file or directory`